tsconfig.json

```diff
@@ -16,15 +16,15 @@
     "incremental": true,
     "plugins": [
       {
-        "name": "next"
-      }
+        "name": "next",
+      },
     ],
     "paths": {
       "@/api/*": ["./app/api/*"],
       "@/assets/*": ["./public/assets/*"],
-      "@/*": ["./*"]
-    }
+      "@/*": ["./*"],
+    },
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "env.mjs"],
-  "exclude": ["node_modules"]
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "exclude": ["node_modules"],
```

Comment:
trailing commas are allowed in json?

Reply:
prettier added these, i was also confused

Reply:
yea ts has custom json format